google/cloud/bigtable/examples/README.md

```diff
@@ -70,12 +82,59 @@ offers a simple way to do so:
 $ gcloud auth login
 ```
 
-## Running the Quick Start Examples using a Cloud Bigtable Instance
+## Samples
+
+### Hello world
+
+View the [Hello World][hello_world_code] example to see sample usage of the Bigtable client library.
+
+#### Usage
 
+```console
+$ ./bigtable_hello_world
+Usage: bigtable_hello_world <project_id> <instance_id> <table_id>
+```
+
+#### Run Hello world
 After configuring gRPC, you can run the examples using:
 
 ```console
-$ export PROJECT_ID=... # The name of your project
-$ export INSTANCE_ID=... # THe name of your instance
+$ export PROJECT_ID=... # Google Cloud Platform project ID
+$ export INSTANCE_ID=... # Cloud Bigtable instance ID
 $ ./bigtable_hello_world ${PROJECT_ID} ${INSTANCE_ID} example-table
 ```
+
+### Administer Instances
+
+View the [example][instance_admin_code] to see sample usage of instance administration of
+the Bigtable client library.
+
+#### Usage
+
+```console
+$ ./bigtable_samples_instance_admin
+
+Usage: bigtable_samples_instance_admin <command> <project_id> [arguments]
+
+Examples:
+  bigtable_samples_instance_admin run my-project my-instance my-cluster us-central1-f
+  bigtable_samples_instance_admin create-dev-instance my-project my-instance us-central1-f
+  bigtable_samples_instance_admin delete-instance my-project my-instance
+  bigtable_samples_instance_admin create-cluster my-project my-instance my-cluster us-central1-a
+  bigtable_samples_instance_admin delete-cluster my-project my-instance my-cluster
+```
+
+#### Run instance admin samples
+
+After configuring gRPC, you can run the examples using:
+
+```console
+$ export PROJECT_ID=... # Google Cloud Platform project ID
+$ export INSTANCE_ID=... # Cloud Bigtable instance ID
+$ export CLUSTER_ID=... # Cloud Bigtable cluster ID
+$ export ZONE=... # The name of zone
```

Comment:
The zone for what?  I would say `# Name of the zone where the example will create a new instance`.

Reply:
fixed